Pre_System_Inerstion.py

```diff
@@ -0,0 +1,17 @@
+import Utilities
+import pandas as pd
+import os
+
+from datetime import datetime, timedelta
+from Member import member_db 
+from Leave import leave_db
+
+data = pd.read_csv(os.getenv("Pre_System_Leaves_excel_Path"))
```

Comment:
can we just make this such that we input a path?

Reply:
Done